crates/ra_syntax/src/ast/extensions.rs

```diff
@@ -234,6 +234,13 @@ impl ast::LetStmt {
             Some(node) => node.kind() == T![;],
         }
     }
+
+    pub fn eq_token(&self) -> Option<SyntaxToken> {
+        self.syntax()
+            .descendants_with_tokens()
```

Comment:
Oh, we should also use children instead of descendants here: descendants is recursive, and so can be much slower. Everything is ok!

bors d+

Reply:
Oh, I see. Is there any documentation for Rowan ?

Reply:
Nope :-( 

Though, the API is stable-ish now, so writing docs should be a good investment. There's api_walkthoruh test which might be helpful.